traefik.go

```diff
@@ -114,13 +137,19 @@ Complete documentation is available at https://traefik.io`,
 
 	traefikConfiguration.ConfigFile = toml.ConfigFileUsed()
 
-	kv, err := CreateKvSource(traefikConfiguration)
+	kv, err = CreateKvSource(traefikConfiguration)
 	if err != nil {
 		fmtlog.Println(err)
 		os.Exit(-1)
 	}
 
-	if kv != nil {
+	usedCmd, err := f.GetCommand()
+	if err != nil {
+		fmtlog.Println(err)
+		os.Exit(-1)
+	}
+	// IF a KV Store is enable and no sub-command called in agrs
```

Comment:
it's the `vim` command to say "replace this with this" 👼
(I know I use `emacs` but :trollface:)